src/components/TilesShared/ImageWithPlaceholder.js

```diff
@@ -9,7 +9,13 @@ const ImageWithPlaceholder = ({ src, ...rest }) => {
   return (
     <React.Fragment>
       {!loaded && <Loader height="40px" width="110px" {...rest} />}
-      <img src={src} onLoad={() => setLoaded(true)} style={{ display: loaded ? 'initial' : 'none' }} {...rest} />
+      <img
+        src={src}
+        data-testid="ImageWithPlaceholder"
```

Comment:
I guess it should be better to use `alt` and later use `ByAltText`.

Reply:
This is purely just for the test of the component itself. In other components, aria labels are passed with src. (ImageWithPlaceholder does not hold any content itself so adding an alt "Image" or something like that would not make much sense)

Reply:
Went trough the usage and looks like every place passes down `alt` so we are good. We might want to consider adding `alt` as required prop. But I'll keep that on you.